roles/openshift_metrics/tasks/install_hawkular.yaml

```diff
@@ -1,73 +1,46 @@
 ---
-- name: Generate hawkular replication controller
-  #template: src=hawkular.j2 dest={{mktemp.stdout}}/templates/metrics-hawkular-rc.yaml
-  template: src=hawkular.j2 dest={{out_dir}}/templates/metrics-hawkular-rc.yaml
-# TODO import templates to this repo?
-- name: create templates
-  command: >
-    {{ openshift.common.client_binary }} -n '{{ project }}'
-    create -f '{{ origin_metrics_dir }}/deployer/templates/{{ item }}.yaml'
+- name: generate replication controller templates
+  template:
+    src: "{{ item }}.j2"
+    dest: "{{ mktemp.stdout }}/templates/{{ item }}.yaml"
   with_items:
-  - hawkular-metrics
-  - hawkular-cassandra
-  - hawkular-cassandra-node-pv
-  - hawkular-cassandra-node-dynamic-pv
-  - hawkular-cassandra-node-emptydir
-  - support
+  - hawkular_metrics_rc
+  - hawkular_cassandra_rc
+- name: test view permission for the hawkular service account
+  command: >
+    {{ openshift.common.client_binary }} get rolebindings
+    -o jsonpath='{.items[?(@.roleRef.name == "view")].userNames}'
+  register: ret
+  changed_when: false
+- name: add the view role to the hawkular service account
+  command: >
+    '{{ openshift.common.admin_binary }}' policy
+    -n '{{ project }}' add-role-to-user view hawkular
+  when: "'system:serviceaccount:{{ project }}:hawkular' not in ret.stdout"
 - name: create hawkular metrics objects
   command: >
     {{ openshift.common.client_binary }} -n '{{ project }}'
-    create -f '{{ mktemp.stdout }}/templates/{{ file }}'
+    create -f '{{ mktemp.stdout }}/templates/{{ item }}'
```

Comment:
I think we are using apply in logging because it will not fail if the object already exists.  Also, do we need to explicitly define the list of object to create versus https://github.com/ewolinetz/openshift-ansible/blob/logging_deployer_tasks/roles/openshift_logging/tasks/main.yaml#L38 which uses a file glob